packages/mui-joy/src/styles/extendTheme.ts

```diff
@@ -453,193 +367,136 @@ export default function extendTheme(themeOptions?: CssVarsThemeOptions): Theme {
+
       lg: `${getCssVar(
         'shadowRing',
         scalesInput.colorSchemes?.light?.shadowRing ?? lightColorSystem.shadowRing,
-      )}, 0.3px 0.8px 1.1px rgba(${getCssVar(
-        'shadowChannel',
-        scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.11), 1.8px 4.5px 6.4px -0.2px rgba(${getCssVar(
+      )}, 0px 2px 8px -2px rgba(${getCssVar(
         'shadowChannel',
         scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.13), 3.2px 7.9px 11.2px -0.4px rgba(${getCssVar(
+      )} / 0.2), 0px 12px 16px -4px rgba(${getCssVar(
         'shadowChannel',
         scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.16), 4.8px 12px 17px -0.5px rgba(${getCssVar(
-        'shadowChannel',
-        scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.19), 7px 17.5px 24.7px -0.7px rgba(${getCssVar(
-        'shadowChannel',
-        scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.21)`,
+      )} / 0.2)`,
+
       xl: `${getCssVar(
         'shadowRing',
         scalesInput.colorSchemes?.light?.shadowRing ?? lightColorSystem.shadowRing,
-      )}, 0.3px 0.8px 1.1px rgba(${getCssVar(
+      )}, 0px 2px 8px -2px rgba(${getCssVar(
         'shadowChannel',
         scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.11), 1.8px 4.5px 6.4px -0.2px rgba(${getCssVar(
+      )} / 0.2), 0px 20px 24px -4px rgba(${getCssVar(
         'shadowChannel',
         scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.13), 3.2px 7.9px 11.2px -0.4px rgba(${getCssVar(
-        'shadowChannel',
-        scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.16), 4.8px 12px 17px -0.5px rgba(${getCssVar(
-        'shadowChannel',
-        scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.19), 7px 17.5px 24.7px -0.7px rgba(${getCssVar(
-        'shadowChannel',
-        scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.21), 10.2px 25.5px 36px -0.9px rgba(${getCssVar(
-        'shadowChannel',
-        scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.24), 14.8px 36.8px 52.1px -1.1px rgba(${getCssVar(
-        'shadowChannel',
-        scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.27), 21px 52.3px 74px -1.2px rgba(${getCssVar(
-        'shadowChannel',
-        scalesInput.colorSchemes?.light?.shadowChannel ?? lightColorSystem.shadowChannel,
-      )} / 0.29)`,
+      )} / 0.2)`,
     },
+
     zIndex: {
       badge: 1,
       table: 10,
       popup: 1000,
       modal: 1300,
       tooltip: 1500,
     },
+
     typography: {
```

Comment:
Which typography level should be the default value? Is it `body-md` or `body-lg`?

Reply:
Maybe we could add `body-xl` and `body-xs` to match the title.

Reply:
Sure, I think it makes sense to add `body-xl` and `body-xs`. I'll do that and review the other `body` values, so we can use `body-md` as the default.

Reply:
From what I tested, I feel like `h1-h4` is still needed because they are 1 to 1 mapping to HTML semantics. Using `<Typography level="h1">` is easier and more intuitive than `<Typogragraphy level="title-lg">` because I don't have to think if I should use `title-lg` or `title-xl` for example.

Another benefit of `h1-h4` is that if you have plain HTML and want to use with Joy UI, you can just replace them like this:
```diff
- <h1>…
+ <Typography level="h1">
- <h2>…
+ <Typography level="h2">
… and so on
```

I think `title-*` and `body-*` are super useful for use in components where sizes are dynamic based on the context that those components are in.

I can combine `title` with `body` like this:

```js
<Typography level="title-md">…</Typography>
<Typography level="body-md">…</Typography>

// or if I need a smaller description
<Typography level="title-md">…</Typography>
<Typography level="body-sm">…</Typography>
```

I put my experiment in Figma https://www.figma.com/file/bRBJTaiU0QYX1MESwApW5D/Joy-UI?type=design&node-id=4177%3A27164&t=lsh6ylB539cnzHEd-1